ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -615,7 +663,7 @@ tests:
       TEST_TYPE: conformance-parallel
       VOLUME_CAPACITY: 64GB
     workflow: openshift-e2e-libvirt-upi
-- as: ocp-e2e-ovn-ppc64le-powervs-original
+- as: ocp-e2e-ovn-powervs-original-multi-p-p
```

Comment:
instead of original, can we say terraform? Or terra for short?
This could be our "deployment type"